src/admin/components/ProCta.vue

```diff
@@ -75,26 +72,30 @@ export default {
     }
 
     .btn {
-        background-color: #fff;
-        color: #f2624d;
-        font-size: 16px;
+        background-color: #A244FF;
+        color: #FFFFFF;
+        font-size: 1.2rem;
         line-height: 1.538em;
         font-weight: 300;
-        border: 1px solid #FF9381;
-        border-radius: 3px;
-        box-shadow: 0px 1px 2px 0px rgba(255, 128, 142, 0.05);
-        display: inline-block;
+        border-radius: 14px;
         text-decoration: none;
-        padding: 12px 27px;
+        display: inline-flex;
+        padding: 16.05px 45.167px;
+        align-items: center;
 
         &:hover {
             box-shadow: 2.736px 7.518px 20px 0 rgba(0,0,0,.15);
```

Comment:
Pls remove the fractional pixel.

Reply:
Bhai , can we use rem instead of pixel ?